src/modm/architecture/interface/can_message.hpp

```diff
@@ -21,26 +22,24 @@
 namespace modm::can
 {
 
-/// Representation of a CAN message
-/// @ingroup modm_architecture_can
-struct Message
+template <uint8_t capacity>
+struct MessageBase
 {
-	inline Message(uint32_t inIdentifier = 0, uint8_t inLength = 0) :
-		identifier(inIdentifier), flags(), length(inLength)
-	{
-	}
+    inline MessageBase(uint32_t inIdentifier = 0, uint8_t inDlc = 0) : 
```

Comment:
Indentation: Here (and in some other places) you used spaces instead of tabs.
modm always uses tabs, please fix the indentation. (Sorry...)

Reply:
Sorry I will fix that